README.md

```diff
@@ -227,6 +227,37 @@ $contractor = $api->get(\Webit\WFirmaSDK\Contractors\ContractorId::create(123));
 
 Feel free to add any other modules support.
 
+## Getting VatCodeId from VAT Rate code by VatCodeIdResolver
+
+### With custom static map provided
+```php
+
+use Webit\WFirmaSDK\Vat\Resolver\VatCodeIdResolverFactory;
+
+$resolver = VatCodeIdResolverFactory::createWithMap(
+  ['23' => 222, '8' => 223] // provide your static ID mapping
+);
+
+$resolver->vatCodeId('23'); // returns new VatCodeId(222);
+```
+
+### With cached map provided by API
```

Comment:
@mgralikowski @jacekkow 
To rozwiązanie daje nam ładne mapowanie bez hardcodowania ID oraz bez pytania API za każdym razem. Cache może być wyczyszczony w przez dewelopera w razie potrzeby (biblioteka na kontroluje cachu).

Reply:
Pytanie kiedy miałbym intencje użyć tego repository, skoro przy dodawaniu itemu do faktury mogę po prostu przekazać stringa `23`? Bo widzę w tej propozycji rozwiązania obsługujesz id jak i kod równolegle.

Reply:
Przy obiekcie `Good` możesz przekazać tylko `VatRate` (który możesz stworzyć ze stringa lub użyć konstruktora statycznego). Jeśli ktoś potrzebuje przekazać jakąś pokręconą stawkę, to tworzy `VatRate` z `VatCodeId`. Repozytorium jest bardziej dla purystów, którzy zawsze chcą przekazywać `VatCodeId`. Nie ma obowiązku go używać - jest po prostu taka opcja.

Reply:
No to dla mnie ok, osobiście repo nie użyje, ale może się komuś przydać. Rozumiem, że mówimy też o Invoice Content?